scripts/generate_scripts_functions.py

```diff
@@ -0,0 +1,338 @@
+import re
```

Comment:
Please add python version information and maybe what the deps are (which can easily be googled or installed through something like pip).
If you ever decide to leave others (we) will have to work with this tool and chances are they (we) are too dumb.